content/support-matrix/delfin/_index.md

```diff
@@ -10,19 +10,20 @@ tags: ["support matrix", "delfin", "drivers"]
-| IBM                                                 | Storewize, SVC            | 6.2–7.8                            | SSH                     | Block   | √                        | √                        | √               | √                    | √         | √           | √               | ×                   |                                                                                                                                                                                                            |
-| Hitachi                                             | VSP                       | 88-01-0X-XX/XX or later            | REST                    | Block   | √                        | √                        | √               | √                    | √         | √           | ×               | ×                   | 1\. The Configuration Manager REST service needs to be installed on the VSP devices.<br>                                                                                                                   |
-| Huawei                                              | OceanStor                | V3/V5                              | REST                    | Unified | √                        | √                        | √               | √                    | √         | √           | √               | ×                   |                                                                                                                                                                                                            |
-| Simulated Sample Driver (for testing the framework) | NA                        | NA                                 | NA                      | NA      | √                        | √                        | √               | √                    | √         | √           | √               | √                   | 1\. Used only for framework testing purpose<br>2\. Performance metric collection is only Alpha version and supports only device level metrics .                                                            |                                                          |
+| vendor                                              | model                     | Version ( Device Manager )         | Access Mechanism (IPV4) | Type    | Basic Device Information | Storage Pool | LUN | Controller | Port | Disk | Qtree | Quota | File system | Share | Capacity Information | SNMP Trap | Alarm Query | Alarm Clearance | Performance Metrics | Remarks/Constraints                                                                                                                                                                                     |   |
```

Comment:
There is an extra column at the end of the table after the `Remarks / Constraints` column. May not be needed. Please check